app/src/main/java/net/squanchy/service/firebase/FirebaseAuthService.java

```diff
@@ -86,6 +86,7 @@ private Completable deleteUser(FirebaseUser user) {
 
     public Completable ifUserSignedInThenCompletableFrom(Func1<String, Completable> completableProvider) {
         return ifUserSignedIn()
+                .firstOrError()
```

Comment:
@fourlastor 
Actually i'm having trouble making this work. After the sign-in signin mechanism was reworked, firstOrError now doesn't propagate any event.